SU-HSDO/suhumsci
# Arch 130 style people list pages

# READY FOR REVIEW

# Summary
Styling the people list view.

# Steps to Test
1. Create some faculty and staff nodes.
1. Create a faculty page and a staff page.
1. Add the Person view to each of these pages, passing "Faculty" or "Staff" arguments to it.
1. Confirm that the theming matches the comp (see below)

# Associated Issues and/or People
- [JIRA ticket](https://stanfordits.atlassian.net/browse/ARCH-130)
- [Staff Comp](https://invis.io/W8L47SJ26JB)
- [Faculty Comp](https://invis.io/KPL47SOWB2S)
